webapp/src/ts/modules/analytics/analytics-target-aggregates-sidebar-filter.component.html

```diff
@@ -0,0 +1,40 @@
+<section class="sidebar-filter">
+  <div class="sidebar-backdrop" [ngClass]="{ hidden: !isOpen }" (click)="toggleSidebarFilter()"></div>
+
+  <div class="sidebar-main" [ngClass]="{ hidden: !isOpen }">
+    <div class="sidebar-header">
+      <p class="sidebar-title">Filter</p>
+      <a class="sidebar-close fa fa-times" (click)="toggleSidebarFilter()"></a>
+    </div>
+
+    <div class="sidebar-body">
+      <mat-accordion multi>
+        <div class="facility-filter-wrap">
+          <div class="facility-filter-title">
+            <span class="facility-title"> {{ facilityLabel | translate }} </span>
+          </div>
+          <div class="facility-filter-body">
+            <div *ngFor="let facility of userFacilities" class="facility-filter-place">
+              <label class="facility-label">
+                <input type="radio" name="facility" [value]="facility._id" class="facility-radio"
+                  [(ngModel)]="selectedFacilityId"
+                  [checked]="facility._id === selectedFacilityId"
+                  (click)="fetchAggregateTargets(facility._id)">
```

Comment:
It's usually every argument in separate lines, but this one has too many. So at least we do the angular ones in separate lines.
```suggestion
                <input type="radio" name="facility" class="facility-radio"
                  [value]="facility._id"
                  [(ngModel)]="selectedFacilityId"
                  [checked]="facility._id === selectedFacilityId"
                  (click)="fetchAggregateTargets(facility._id)">
```
Question: Are you sure you need `[value]="facility._id"`? The ` [(ngModel)]="selectedFacilityId"` is double-way binding, so for default values, it's set in the ngModel, and the value isn't needed.

Reply:
I think we still need value to know when the facility is changed. What it seems we don't need is the `[checked]` . From what I have read, `ngModel` handles the checked state.